docset/windows/iisadministration/get-iisconfigattributevalue.md

```diff
@@ -48,7 +48,7 @@ This command gets the configuration attribute MaxUrlSegments for the Default Web
 ### Example 2: Get configuration attribute values from different parts of the configuration store
 ```
 PS C:\> Get-IISConfigSection "system.webServer/asp" | Get-IISConfigAttributeValue -AttributeName "ScriptErrorMessage"
-error message inside the location path default web site
+An error occurred on the server when processing the URL. Please contact the system administrator. <p/> If you are the system administrator please click <a href="http://go.microsoft.com/fwlink/?LinkID=82731">here</a> to find out more about this error.
```

Comment:
```suggestion
An error occurred on the server while processing the URL. Please contact the system administrator. <p/> If you are the system administrator, please click <a href="http://go.microsoft.com/fwlink/?LinkID=82731">here</a> to find out more about this error.
```

Reply:
This message is actually returned by the command so I have copied it and pasted in the article. If we want to edit it we need to edit the code of IIS I guess.